Clinical-Genomics/scout
# Delete Unused File: scout/server/userpanel.py

Ref: #2263
This PR adds refactoring by deleting used file `scout/server/userpanel.py`.

**How to test**:
1. Install
2. No changes should be noticed (deleted unused file)
3. Pytests should execute 

**Expected outcome**:
The functionality should be working

**Review:**
